dev-client/src/model/soilId/actions/soilDataActionFields.ts

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright © 2024 Technology Matters
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/.
+ */
+
+import {
+  DepthDependentSoilDataUpdateMutationInput,
+  SoilDataUpdateDepthIntervalMutationInput,
+  SoilDataUpdateMutationInput,
+} from 'terraso-client-shared/graphqlSchema/graphql';
+
+import {
+  DepthDependentSoilData,
+  SoilData,
+  SoilDataDepthInterval,
+} from 'terraso-mobile-client/model/soilId/soilIdSlice';
+
+/**
+ * The soil data fields which are covered by the update action.
+ */
+export const SOIL_DATA_UPDATE_FIELDS = [
```

Comment:
I am open to a better means of enumerating these fields in the future (via some sort of codegen?), but this was the best option that the other devs and I could come up with.